### DISCUSSION TITLE
Plans for upgrading dependencies?

### DISCUSSION THREAD

**Discussion by jorqensen** [tighten/jigsaw#577] [Q&A] [answered]
Not sure if this should be an issue or discussion, but is there any plans on upgrading for the `Illuminate` components to 8.x out of the box so we can use directives like `@class` and so on? 🤔

**Comment by bakerkretzmar** [comment, marked as answer]
Probably not in the near future, no, see #571. We're most likely going to wait until Jigsaw v2.
